ttnn/cpp/ttnn/operations/conv/conv2d/conv2d.cpp

```diff
@@ -204,13 +204,13 @@ OptimizedConvParallelizationConfig determine_conv_op_parallel_config_from_conv_o
     TT_ASSERT(conv_output_mem_config.shard_spec.has_value());
     const auto& shard_spec = conv_output_mem_config.shard_spec.value();
     const auto& shard_shape = shard_spec.shape;
-    TT_ASSERT(shard_shape[0] % 32 == 0);
+    // TT_ASSERT(shard_shape[0] % 32 == 0);
```

Comment:
delete

Reply:
does this affect height and block sharding variants?

Reply:
Fixed.